interfaces/policy/policy.go

```diff
@@ -116,22 +117,49 @@ func (ic *InstallCandidate) Check() error {
 
 // ConnectCandidate represents a candidate connection.
 type ConnectCandidate struct {
-	// TODO: later we need to carry dynamic attributes once we have those
-	Plug                *snap.PlugInfo
+	Plug *interfaces.ConnectedPlug
+	// Static + dynamic attributes, merged lazily by plugAttrs below (FIXME: remove it)
+	PlugAttrs           map[string]interface{}
```

Comment:
does this need to be public given that is a cache and will go away?

Reply:
Done.